docs/changelog.md

```diff
@@ -283,6 +283,17 @@ future and `StringSlice.__len__` now does return the Unicode codepoints length.
   # pw_gecos='System Administrator', pw_dir='/var/root', pw_shell='/bin/zsh')
   ```
 
+- Added `SIMD.count` method for `Scalar` rhs.
+  ([PR #3179](https://github.com/modularml/mojo/pull/3179) by [@rd4com](https://github.com/rd4com))
+
+  Example usage:
+
+  ```mojo
+  var vector_of_four_int64 = SIMD[DType.int64, 4](1)
+  vector_of_four_int64[0] = 0
+  print(vector_of_four_int64.count(1)) # 3
```

Comment:
Hmm, maybe use here the same example what's below? Instead of these 3 lines above:
`print(SIMD[DType.uint8, 4](1, 1, 1, 0).count(1)) # 3`

Reply:
Yes @gryznar , new commit implement this suggestion :+1: (https://github.com/modularml/mojo/pull/3179/commits/87761d025c82d991e55fba7f1ce3cf8c1bbcff1c)